src/test/java/com/newworld/saegil/authentication/service/AuthenticationServiceTest.java

```diff
@@ -188,21 +176,25 @@ class Context_with_invalid_token {
                 // given
                 final User user = new User("권예진", "http://example.com/profile.jpg", "1234567", OAuth2Type.KAKAO);
                 userRepository.save(user);
-                final Token token = tokenProcessor.generateToken(LocalDateTime.now(), new PrivateClaims(user.getId()).toMap());
+                final Token token = tokenProcessor.generateToken(LocalDateTime.now(),
+                        new PrivateClaims(user.getId()).toMap());
```

Comment:
이 부분도 마찬가지 입니다. 그래서 개행을 한다면 아래처럼 하는 것을 더 선호합니다.
```suggestion
                final Token token = tokenProcessor.generateToken(
                        LocalDateTime.now(),
                        new PrivateClaims(user.getId()).toMap()
                );
```